sv.h

```diff
@@ -2242,7 +2242,7 @@ immediately written again.
         sv_setsv_flags(dsv, ssv, SV_GMAGIC|SV_DO_COW_SVSETSV)
 /*
 =for apidoc_defn Am|void|sv_setsv_nomg|SV *dsv|SV *ssv
-=for apidoc_defn Am|void|sv_catsv_nomg|SV *dsv|SV *ssv
+=for apidoc_defn Am|void|sv_catsv_nomg|SV * const dsv|SV * const sstr
```

Comment:
I don't understand the changes in the "fifth column" in line 2245.  Why does `SV *ssv` get changed to `SV * const sstr`?

Reply:
Because all the other forms of `sv_catsv` have that as their parameter.  `sv_setsv` forms use `ssv`.

If you look at the generated `pod/perlapi.pod`, the entries for both groups now are internally consistent after this change.